native/src/test/java/io/ballerina/stdlib/http/api/TestUtils.java

```diff
@@ -37,6 +37,8 @@
  */
 public class TestUtils {
 
+    public static final int SOCKET_SERVER_PORT = 9001;
```

Comment:
```suggestion
    public static final int SOCKET_SERVER_PORT = 8001;
```

Reply:
Let's use 8000 range ports at native test level. So it does not conflict with ballerina tests